### DISCUSSION TITLE
export / migrate data

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#8587] [Unsupported Versions] [answered]
I have an existing distributed 2.3.130 install that I want to wipe and do a fresh install to ver 2.3.150.  Basically I dorked up my install a bit and I would like to get it back to a clean state.  But, I don't want to lose the log data I have.  

Is there a way I can export or migrate the data to another SO instance?  I did some searching and I didn't find anything.

**Comment by dougburks** [comment, marked as answer]
Here are some links that might help:
https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html
https://github.com/elasticsearch-dump/elasticsearch-dump
